static/script/devices/browserdevice.js

```diff
@@ -560,7 +560,7 @@ define(
              * @returns {antie.Historian} an object that can be used to get a back or forward url between applications while preserving history
              */
             getHistorian: function getHistorian () {
-                return new Historian(decodeURI(this.getWindowLocation().href));
+                return new Historian(decodeURI(this.getWindowLocation().href), this.getCurrentAppURLParams());
```

Comment:
Are you sure getCurrentAppURLParams() works?
I think exists in the application.js and not `this`.

Reply:
Sorry, yes. This should be self._application.getCurrentAppURLParams() (I was stepping through application, device and browserdevice and must've got confused.)